app/views/acceptance/accept.erb

```diff
@@ -0,0 +1,6 @@
+Congratulations <%= @student_application.first_name %>,
+
+You have been accepted to the Rails Decal!
+
+Thanks,
+Rails Decal Staff
```

Comment:
I didn't know you could omit the `.html` part of the email. I'm assuming it still sends correctly, haha.